packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3762,16 +3738,19 @@ export class ContainerRuntime
 	}
 
 	/**
-	 * Fetches the latest snapshot from storage and uses it to refresh SummarizerNode's
-	 * internal state as it should be considered the latest summary ack.
+	 * Fetches the latest snapshot from storage to refresh the cache as a performance optimization and closes the
+	 * summarizer to reload from new state.
 	 * @param summaryLogger - logger to use when fetching snapshot from storage
-	 * @returns downloaded snapshot's reference sequence number
+	 * @returns a generic summarization error
 	 */
-	private async refreshLatestSummaryAckFromServer(
+	private async prefetchLatestSummaryThenClose(
```

Comment:
Do you want to unify this with the code inside the if block above on line 3695? Where it also calls this.fetchSnapshotFromStorage then closes.  It has an extra check that this doesn't, so it's not totally clear cut, but curious if you've considered it.

Reply:
I think that there's enough code difference between the two that they should be slightly different methods. The parts that are similar have been abstracted out. I think I originally tried, but I felt like the code was harder to read.